python/triton/runtime/jit.py

```diff
@@ -838,6 +841,7 @@ def jit(
            * python primitives,
            * builtins within the triton package,
            * arguments to this function,
+           * meta-parameters through keyword arguments or the :code:`META` dictionary argument,
```

Comment:
This doesn't quite seem like the right place to document this, since it's not the JITFunction itself that receives meta parameters, but the callable passed as an argument to a JITFunction's __getitem__ method (the `grid` lambda).  Also `META` is just sort of a convention, we pass a dictionary into `grid` containing the argument values

Reply:
Yeah, I couldn't figure out a better place for documentation. Also `grid` seems not be documented anywhere.